packages/datadog_flutter_plugin/lib/src/tracing/tracing_headers.dart

```diff
@@ -75,13 +87,16 @@ class TracingUUID {
         break;
     }
 
-    return null;
+    return TracingUUID(BigInt.zero);
   }
 
   String asString(TraceIdRepresentation representation) {
     switch (representation) {
       case TraceIdRepresentation.decimal:
         return value.toString();
+      case TraceIdRepresentation.decimal63bit:
```

Comment:
I wonder if it makes sense to have `decimal_X_bit` representation. For header values I think we're want:
decimal, hexadecimal16Chars, hexadecimal32Chars. The fact that it's an integer from 1 to 2^63 is UUID generation implementation detail.

Reply:
This was because I thought we should be tuncating. I'll remove it.